diff --git a/build/src/start/server/metro/metroWatchTypeScriptFiles.js b/build/src/start/server/metro/metroWatchTypeScriptFiles.js
index 3c4e29a3518d0600ab4f9b280aa3d196aa4468a6..e17fc961f4a08f60d33954ea69b6c608b8d0b271 100644
--- a/build/src/start/server/metro/metroWatchTypeScriptFiles.js
+++ b/build/src/start/server/metro/metroWatchTypeScriptFiles.js
@@ -28,7 +28,7 @@ function metroWatchTypeScriptFiles({ metro, server, projectRoot, callback, tscon
 ] }) {
     const watcher = metro.getBundler().getBundler().getWatcher();
     const tsconfigPath = _path().default.join(projectRoot, 'tsconfig.json');
-    const listener = ({ eventsQueue })=>{
+    const listener = ({ eventsQueue = [] })=>{
         for (const event of eventsQueue){
             var _event_metadata;
             if (eventTypes.includes(event.type) && ((_event_metadata = event.metadata) == null ? void 0 : _event_metadata.type) !== 'd' && // We need to ignore node_modules because Metro will add all of the files in node_modules to the watcher.
diff --git a/build/src/start/server/metro/waitForMetroToObserveTypeScriptFile.js b/build/src/start/server/metro/waitForMetroToObserveTypeScriptFile.js
index 87cdf53ef15d6f4d72f4e310aea0a90bfaeda3c4..5ec6e7160e2562a78e1c51e951c3385272430291 100644
--- a/build/src/start/server/metro/waitForMetroToObserveTypeScriptFile.js
+++ b/build/src/start/server/metro/waitForMetroToObserveTypeScriptFile.js
@@ -35,7 +35,7 @@ const debug = require('debug')('expo:start:server:metro:waitForTypescript');
 function waitForMetroToObserveTypeScriptFile(projectRoot, runner, callback) {
     const watcher = runner.metro.getBundler().getBundler().getWatcher();
     const tsconfigPath = _path().default.join(projectRoot, 'tsconfig.json');
-    const listener = ({ eventsQueue })=>{
+    const listener = ({ eventsQueue = [] })=>{
         for (const event of eventsQueue){
             var _event_metadata;
             if (event.type === 'add' && ((_event_metadata = event.metadata) == null ? void 0 : _event_metadata.type) !== 'd' && // We need to ignore node_modules because Metro will add all of the files in node_modules to the watcher.
@@ -63,7 +63,7 @@ function waitForMetroToObserveTypeScriptFile(projectRoot, runner, callback) {
 }
 function observeFileChanges(runner, files, callback) {
     const watcher = runner.metro.getBundler().getBundler().getWatcher();
-    const listener = ({ eventsQueue })=>{
+    const listener = ({ eventsQueue = [] })=>{
         for (const event of eventsQueue){
             var // event.type === 'add' &&
             _event_metadata;
@@ -89,7 +89,7 @@ function observeFileChanges(runner, files, callback) {
 }
 function observeAnyFileChanges(runner, callback) {
     const watcher = runner.metro.getBundler().getBundler().getWatcher();
-    const listener = ({ eventsQueue })=>{
+    const listener = ({ eventsQueue = [] })=>{
         callback(eventsQueue);
     };
     watcher.addListener('change', listener);
